Doc/conf.py

```diff
@@ -55,6 +56,12 @@
 
 manpages_url = 'https://manpages.debian.org/{path}'
 
+# Intersphinx configuration
+intersphinx_mapping = {
+    'typing': ('https://typing.readthedocs.io/en/latest/', None),
```

Comment:
Still pending a discussion/decision about offline builds, but:
```suggestion
    'typing': ('https://typing.python.org/en/latest/', None),
```